### PR TITLE
Run link validator and release drafter (as GitHub workflows)

### PR DESCRIPTION
**Pull request by ennru**

### CODE REVIEW

File: .github/workflows/link-validator.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
+name: Link Validator
+
+on:
+  pull_request_target:
+    branches: [$default-branch]
+  schedule:
+    - cron:  '0 6 * * 1'
```
Comment (seglo) [inline .github/workflows/link-validator.yml:7]: How come not on a PR?

**Review by seglo — COMMENTED** [review]
Interesting. I like the approach of slowly transitioning to actions.

File: .github/workflows/link-validator.yml (hunk 1 above)
Comment (ennru) [reply .github/workflows/link-validator.yml:7]: `pull_request_target` means "run on pull requests targeting the listed branches".

Comment (seglo) [reply .github/workflows/link-validator.yml:7]: Got it.  Is it also running on a cron?

Comment (ennru) [reply .github/workflows/link-validator.yml:7]: Yes, every gardening Monday.

**Review by seglo — APPROVED** [review]

**Pull request merged**